README.md

```diff
@@ -18,210 +18,181 @@ Darkflame Universe is licensed under AGPLv3, please read [LICENSE](LICENSE). Som
+# Prepare for installation
+sudo apt update && sudo apt install -y software-properties-common lsb-release && sudo apt clean all
 
-:: Run CMake with build flag to build
-cmake --build . --config Release
-```
+# Obtain a copy of the signing key
+wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | sudo tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null
 
-### Windows builds (WSL)
-This section will go through how to install [WSL](https://docs.microsoft.com/en-us/windows/wsl/install) and building in a Linux environment under Windows. WSL requires Windows 10 version 2004 and higher (Build 19041 and higher) or Windows 11.
+# Add the repository to your sources list.
+sudo apt-add-repository "deb https://apt.kitware.com/ubuntu/ $(lsb_release -cs) main"
 
-#### Open the Command Prompt application with Administrator permissions and run the following:
-```bash
-# Installing Windows Subsystem for Linux
-wsl --install
-```
+# Next you'll want to ensure that Kitware's keyring stays up to date
+sudo apt update
+sudo apt install kitware-archive-keyring
+sudo rm /etc/apt/trusted.gpg.d/kitware.gpg
 
-#### Open the Ubuntu application and run the following:
-```bash
-# Make sure the install is up to date
-apt update && apt upgrade
+# If sudo apt update above returned an error, copy the public key at the end of the error message and run the following command
+# if the error message was "The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 6AF7F09730B3F0A4"
+# then the below command would be "sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6AF7F09730B3F0A4"
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys <TheCopiedPublicKey>
 
-# Make sure the gcc, cmake, and build-essentials are installed
-sudo apt install gcc
+# Finally update and install
+sudo apt update
 sudo apt install cmake
-sudo apt install build-essential
 ```
 
-[**Follow the Linux instructions**](#linux-builds)
+## Database setup
+First you'll need to start MariaDB.
 
-### ARM builds
-AArch64 builds should work on linux and MacOS using their respective build steps. Windows ARM should build but it has not been tested
+For Windows the service is always running by default.
 
-### Updating your build
-To update your server to the latest version navigate to your cloned directory
+For MacOS, run the following command
 ```bash
-cd /path/to/DarkflameServer
+brew services start mariadb
 ```
-run the following commands to update to the latest changes
+
+For Linux, run the following command
 ```bash
-git pull
-git submodule update --init --recursive
+sudo systemctl start mysql
+# If systemctl is not a known command on your distribution, try the following instead
+sudo service mysql start
 ```
-now follow the build section for your system
 
-## Setting up the environment
+<font size="4">**You will need to run this command every time you restart your environment**</font>
```

Comment:
If you change the above systemctl command to `sudo systemctl enable --now mysql` it will make it startup automatically on system restart.

Not sure about `sudo service mysql start`  though

Reply:
Not sure if that's desirable - if they don't turn it off, it could use resources without them knowing. If it gets included, both should be given with that explanation

Reply:
Hmm fair point, maybe added as a note or something

Reply:
I too am not a fan of unknowingly background running applications so making this an opt in rather than an opt out would be ideal